include javascript files client side, not from Smarty.
as i've separated very clearly files available as htdocs and files that are not.

and besides, js files should be (can be) cached at client side, which is good.

glen 2005-07-30, 2005-11-30

--- ./js/expandable_cell.js	2005-07-30 17:47:12.000000000 +0300
+++ /tmp/expandable_cell.js	2005-07-30 20:18:01.000000000 +0300
@@ -1,5 +1,3 @@
-{literal}
-
 var expanding = false;
 
 // expands the cell specified by ecID and msgID. This will initiate the call to the remote script to get the
@@ -97,4 +95,3 @@
     }
     return newCells;
 }
-{/literal}
\ No newline at end of file
--- ./js/httpclient.js	2005-07-30 17:47:12.000000000 +0300
+++ /tmp/httpclient.js	2005-07-30 20:18:01.000000000 +0300
@@ -1,5 +1,3 @@
-{literal}
-
 function HTTPClient() {};
 
 HTTPClient.prototype = {
@@ -49,4 +47,3 @@
         }
     }
 }
-{/literal}
\ No newline at end of file
--- ./templates/view.tpl.html	2005-07-30 17:47:13.000000000 +0300
+++ /tmp/view.tpl.html	2005-07-30 20:18:01.000000000 +0300
@@ -1,9 +1,7 @@
 {include file="header.tpl.html" extra_title=$extra_title}
 {include file="navigation.tpl.html"}
-<script language="javascript">
-{include file="js/httpclient.js"}
-{include file="js/expandable_cell.js"}
-</script>
+<script src="{$rel_url}js/httpclient.js"></script>
+<script src="{$rel_url}js/expandable_cell.js"></script>
 {if $issue == ""}
   <table width="400" align="center">
     <tr>
--- ./templates/mail_queue.tpl.html	2005-07-30 17:47:13.000000000 +0300
+++ /tmp/mail_queue.tpl.html	2005-07-30 20:21:46.000000000 +0300
@@ -3,10 +3,8 @@
 {if $denied == 1}
     <div class="default" align="center">Sorry, you do not have permission to view this page</div>
 {else}
-<script language="javascript">
-{include file="js/httpclient.js"}
-{include file="js/expandable_cell.js"}
-</script>
+<script src="{$rel_url}js/httpclient.js"></script>
+<script src="{$rel_url}js/expandable_cell.js"></script>
 <form>
   <table bgcolor="#FFFFFF" width="100%" cellspacing="1" cellpadding="2" border="0">
     <tr>
--- eventum-localization-dev2/templates/reports/recent_activity.tpl.html~	2006-06-02 01:01:26.205901620 +0300
+++ eventum-localization-dev2/templates/reports/recent_activity.tpl.html	2006-06-02 01:02:30.727354382 +0300
@@ -1,9 +1,7 @@
 {capture assign="extra_title"}{t}Recent Activity{/t}{/capture}
 {include file="header.tpl.html" extra_title=$extra_title}
-<script language="javascript">
-{include file="../js/httpclient.js"}
-{include file="../js/expandable_cell.js"}
-</script>
+<script language="javascript" src="{$rel_url}js/httpclient.js"></script>
+<script language="javascript" src="{$rel_url}js/expandable_cell.js"></script>
 <br />
 <form method="post" action="{$smarty.server.PHP_SELF}" name="recent_activity">
 <input type="hidden" name="cat" value="generate">
--- eventum/templates/update.tpl.html~	2007-03-08 19:56:02.839767679 +0200
+++ eventum/templates/update.tpl.html	2007-03-08 19:56:04.349801696 +0200
@@ -1,8 +1,6 @@
 {include file="header.tpl.html"}
 {include file="navigation.tpl.html"}
-<script language="javascript">
-{include file="js/httpclient.js"}
-</script>
+<script src="{$rel_url}js/httpclient.js"></script>
 
 {if $issue == ""}
   <table width="300" align="center">
